releasenotes/notes/port-elide-permutations-ed91c3d9cef2fec6.yaml

```diff
@@ -3,7 +3,3 @@ features_transpiler:
   - |
     Port most of the logic of the transpiler pass :class:`~.ElidePermutations`
     to Rust.
-fixes:
```

Comment:
Is there a reason to drop the message you'd added here?

Reply:
Yes, we have decided to put the bug fix into a separate python-only PR so that it could be backported (please see #13186).